angular-spa/README.md

```diff
@@ -0,0 +1,109 @@
+- ms-graph
+urlFragment: ms-identity-docs-javascript-angular-spa
+---
+-->
+
+# Angular | web | user sign-in, protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This Angular application uses the Angular MSAL v2 library to authenticate a user and then makes a request to the Graph API as the authenticated user. The response to the request is displayed in the web browser.
+
+![Screenshot with profile information from Graph](profile.png?raw=true)
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Node.js 16+
+- Angular 12
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the application.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `Angular SPA`                                                                | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | Single-Page Application                                                      | Required value for this sample.                                                 |
+| **Redirect URI**                  | `http://localhost:4200`                                                      | Required value for this sample.                                                                    |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample in _app.module.ts_ with app registration values
+
+```javascript
+// 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+clientId: ''
+
+// Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+authority: ''
```

Comment:
I have removed this in my suggestion last because I wasn't thinking that the redirectUri would end up being necessary.  Whoops.  Adding back in for you :)

```suggestion
authority: '',
```